heartbeat/monitors/active/dialchain/net.go

```diff
@@ -45,7 +45,7 @@ import (
 //    }
 //  }
 func TCPDialer(to time.Duration) NetDialer {
```

Comment:
nit: I see that the file `net.go` contains only dialers. Do you think we can rename it to `dialers.go`?
Also, there are some functions in `util.go` that refer to `transport.Dialer`, I wonder if we can divide the stuff in  these two files into `dialers.go` and `layers.go`. We could get rid of utils ;)

Reply:
Yep, good call.